app/models/card.rb

```diff
@@ -6,6 +6,20 @@ class Card < ApplicationRecord
 
   REVIEW_TIME_GAP = 3
 
+  def self.random
+    return nil if due_today.empty?
```

Comment:
@kennyfrc left this line. If ActiveRecord will not receive the data it will return empty array by default